ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcErrorConverter.java

```diff
@@ -44,7 +44,7 @@ public static JsonRpcError convertTransactionInvalidReason(
       case GAS_PRICE_TOO_LOW:
         return JsonRpcError.GAS_PRICE_TOO_LOW;
       case PRIVACY_GROUP_DOES_NOT_EXIST:
```

Comment:
NIT: If we are renaming the JsonRpcError, should we also rename this enum value to be consistent?

Reply:
Done